src/views/TableDetail.vue

```diff
@@ -13,34 +16,41 @@
             All Tables
           </v-subheader>
 
-          <v-divider></v-divider>
+          <v-divider />
 
           <v-list-item
+            v-for="t in tables"
+            :key="t"
             ripple
-            v-for="table in tables"
-            :key="table"
```

Comment:
This change makes the code slightly less readable on first glance

Reply:
I agree, and the linter seems fine with the order of:

```
v-for
:key
:to
ripple
```

So my vote is for that

Reply:
I meant using t instead of table here. I'm okay with any order, really 😄

Reply:
I had to use something besides `table` to avoid shadowing the `table` variable declared in the containing scope. Iteration using a pattern like `t in tables` is pretty idiomatic so let's leave it.